core/events/domain/event.py

```diff
@@ -377,6 +392,10 @@ class UpdateAuthors(Event):
 
     authors: List[Author] = field(default_factory=list)
```

Comment:
Should the submission API continue to accept the classic "raw authors string", at least for a time? Otherwise we are imposing our formatting on the author display when we interpret the list.

Reply:
Yeah, that's good point. Many journals allow a custom display name on a per-author basis. What if we were to support a ``display`` field on ``Author``, and then assemble the canonical author string using those values? That way we can move forward on structured authors, but still provide flexibility?

Of course, I can be persuaded to just keep the freeform field for now.